src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -1450,14 +1501,15 @@ class ContentSidebar extends PureComponent<Props, State> {
             activityFeedError,
             approverSelectorContacts,
             mentionSelectorContacts,
-            currentUserError
+            currentUserError,
+            isFileLoading
         }: State = this.state;
 
         const styleClassName = classNames(
             'be bcs',
             {
                 [`bcs-${view}`]: !!view,
-                'bcs-is-open': !!view
+                'bcs-is-open': !!view || !isCollapsed
```

Comment:
make sure to test responsive mode, shrinking viewport

Reply:
Seems good, collapses when under a certain amount of px